src/test/java/core/basesyntax/service/FruitTransactionMapperTest.java

```diff
@@ -0,0 +1,47 @@
+package core.basesyntax.service;
+
+import static junit.framework.TestCase.assertNotNull;
+import static org.junit.Assert.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.model.FruitTransaction;
+import java.util.Arrays;
+import java.util.List;
+import org.junit.jupiter.api.Test;
+
+public class FruitTransactionMapperTest {
+
+    @Test
+    public void mapData_NullData_Check_ThrowsRuntimeException_NotOk() {
```

Comment:
```suggestion
    public void mapData_nullData_throwsRuntimeException() {
```

Reply:
let's follow the suggestd naming convention - `<method name> _ < conditions > _ <expected result>`